code/datums/tutorial/xenomorph/_xenomorph.dm

```diff
@@ -0,0 +1,28 @@
+/datum/tutorial/xenomorph
+	category = TUTORIAL_CATEGORY_XENO
+	parent_path = /datum/tutorial/xenomorph
+	icon_state = "marine" // CHANGE
+	///Starting xenomorph type (caste) of type /mob/living/carbon/xenomorph/...
+	var/mob/living/carbon/xenomorph/starting_xenomorph_type = /mob/living/carbon/xenomorph/drone
+	///Reference to the actual xenomorph mob
+	var/mob/living/carbon/xenomorph/xeno
+
+/datum/tutorial/xenomorph/init_mob()
+	var/mob/living/carbon/xenomorph/new_character = new starting_xenomorph_type(bottom_left_corner, null, XENO_HIVE_TUTORIAL)
+	new_character.lastarea = get_area(bottom_left_corner)
+
+	//Remove all actions by default, we will give actions to the player when needed.
+	for(var/datum/action/action_path as anything in new_character.base_actions)
```

Comment:
Make this optional in case we don't want to do this for every tutorial.

Reply:
I'd say make it a simple boolean on /datum/tutorial/xenomorph; if people want more granularity than that they can just override init_mob()